### ISSUE TITLE
add rake task jobshop:sites:show

### ISSUE THREAD

**Issue by frankjmattia** [jobshop/jobshop#18]
should output something along the lines of:

```
id                                         name                 email
==                                         ====                 =====
39d8fcfa-e03f-4565-bf18-dc4714a47452   |   My Jobshop       |   johnqvaxy@example.com
```

**Comment by frankjmattia** [comment]
now called rake `jobshop:teams`. closed by 4936325557915bd47f735f4ea8b7203f8fac96aa

**Comment by frankjmattia** [comment]
Shouldn't need this once there's a team admin interface... basically just for quick debugging
